test/conformance/helpers/tracing/zipkin.go

```diff
@@ -47,10 +49,15 @@ var setTracingConfigOnce = sync.Once{}
 // TODO Do we need a tear down method to revert the config map to its original state?
 func setTracingConfigToZipkin(t *testing.T, client *lib.Client) {
 	setTracingConfigOnce.Do(func() {
-		err := client.Kube.UpdateConfigMap("knative-eventing", "config-tracing", map[string]string{
-			"backend":         "zipkin",
-			"zipkin-endpoint": "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans",
```

Comment:
https://github.com/knative/eventing/blob/622eafe6e29cdf08cbe31550d9dc87cbbc1cf793/pkg/tracing/setup.go#L127

should the above be make more generic as well, instead of hard-coded?

Reply:
OK, looks like @slinkydeveloper is doing #2978